src/NuGet.Core/NuGet.Commands/NuGet.Commands.csproj

```diff
@@ -4,10 +4,10 @@
 
   <PropertyGroup>
     <Description>Complete commands common to command-line and GUI NuGet clients.</Description>
-    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworksLibrary);netstandard2.1</TargetFrameworks>
     <TargetFramework />
     <NoWarn>$(NoWarn);CS1591;CS1574;CS1573;CS1584;CS1658</NoWarn>
-    <NoWarn Condition="'$(TargetFramework)' == '$(NetStandardVersion)'">$(NoWarn);CS1998</NoWarn>
+    <NoWarn Condition="$(TargetFramework.StartsWith('netstandard'))">$(NoWarn);CS1998</NoWarn>
```

Comment:
IsCore.

Reply:
As we discussed, the IsCore is in common.targets,  NoWarn could not get the right value if changing the condition to IsCore.